indra/newview/scripts/lua/require/UI.lua

```diff
@@ -169,4 +173,61 @@ function UI.addMenuSeparator(...)
     return leap.request('UI', args)
 end
 
+-- ***************************************************************************
+--  Toolbar buttons
+-- ***************************************************************************
+-- Clears all buttons off the toolbars
+function UI.clearToolbars()
+    leap.send('UI', {op='clearToolbars'})
+end
+
+function UI.defaultToolbars()
+    leap.send('UI', {op='defaultToolbars'})
+end
+
+-- UI.addToolbarBtn{btn_name=btn_name
+--          [, toolbar= 3] -- 1 [TOOLBAR_LEFT], 2 [TOOLBAR_RIGHT], 3 [TOOLBAR_BOTTOM]
+--          [, rank=1]} -- position on the toolbar
```

Comment:
but `rank=1` would put the new button in the second position, yes?

Reply:
Yes, that's right. I'll mention it here.